Documentation/Troubleshooting/disaster-recovery.md

```diff
@@ -276,3 +276,93 @@ and deleting the other deployments. An example command to do this is `k -n rook-
 1. Copy the endpoints configmap from the old cluster: `rook-ceph-mon-endpoints`
 1. Scale the rook operator up again : `kubectl -n rook-ceph scale deployment rook-ceph-operator --replicas 1`
 1. Wait until the reconciliation is over.
+
+## Restoring the Rook cluster after the Rook namespace is deleted
+
+When the rook-ceph namespace is accidentally deleted, the good news is that the cluster can be restored. With the content in the directory `dataDirHostPath` and the original OSD disks, the ceph cluster could be restored with this guide.
+
+You need to manually create a ConfigMap and a Secret to make it work. The information required for the ConfigMap and Secret can be found in the `dataDirHostPath` directory.
+
+The first resource is the secret named `rook-ceph-mon` as seen in this example below:
+
+```console
+apiVersion: v1
+data:
+  ceph-secret: QVFCZ0h6VmorcVNhSGhBQXVtVktNcjcrczNOWW9Oa2psYkErS0E9PQ==
+  ceph-username: Y2xpZW50LmFkbWlu
+  fsid: M2YyNzE4NDEtNjE4OC00N2MxLWIzZmQtOTBmZDRmOTc4Yzc2
+  mon-secret: QVFCZ0h6VmorcVNhSGhBQXVtVktNcjcrczNOWW9Oa2psYkErS0E9PQ==
+kind: Secret
+metadata:
+  finalizers:
+  - ceph.rook.io/disaster-protection
+  name: rook-ceph-mon
+  namespace: rook-ceph
+  ownerReferences: null
+type: kubernetes.io/rook
+
+```
+
+The values for the secret can be found in `$dataDirHostPath/rook-ceph/client.admin.keyring` and `$dataDirHostPath/rook-ceph/rook-ceph.config`.
+- `ceph-secret` and `mon-secret` are to be filled with the `client.admin`'s keyring contents.
+- `ceph-username`: set to the string `client.admin`
+- `fsid`: set to the original ceph cluster id.
+All the fields in data section need to be encoded in base64. Coding could be done like this:
```

Comment:
Blank line before and after a paragraph, otherwise the rendering combines this with the previous bullet.